sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -189,9 +188,15 @@ bool DistortionCorrector<T>::isInputValid(sensor_msgs::msg::PointCloud2 & pointc
 }
 
 template <class T>
-bool DistortionCorrector<T>::azimuthConversionExists(sensor_msgs::msg::PointCloud2 & pointcloud)
+std::optional<AngleConversion> DistortionCorrector<T>::tryComputeAngleConversion(
+  sensor_msgs::msg::PointCloud2 & pointcloud)
 {
-  if (!isInputValid(pointcloud)) return false;
+  // This function try to compute the angle conversion from Cartesian coordinates to LiDAR azimuth
```

Comment:
```suggestion
  // This function tries to compute the angle conversion from Cartesian coordinates to LiDAR azimuth
```
Also, isn't it lidar to cartesian?